.dockerignore

```diff
@@ -1 +1,3 @@
 data
+.git
```

Comment:
this doesn't work for the garage-dev target though, right? or does COPY not obey .dockerignore

Reply:
COPY obeys `.dockerignore`, but we need the `.git` folder since there are some tests which need it, the ones that test the archival of source code inside the experiment directory. So I don't have `.git` in `.dockerignore` anymore